src/runtime_src/core/tools/common/ReportAsyncError.cpp

```diff
@@ -57,8 +93,6 @@ populate_async_error(const xrt_core::device * device)
     // ignore, likely that async error not supported
   }
 
-  pt.add_child("errors", error_array);

```

Comment:
The JSON schema has changed rather drastically, (e.g., removal of the errors array, asynchronous_errors becoming and array (from originally being a dictionary)).  

It "might" be O.K. for there most likely might not be any customers using this schema.  Never the less, it should be discussed at the XRT PRT.

Reply:
Chatted with Sarabjeet off-line.  This schema was introduced in 2021.2, so schema changes don't require XRT PRT approval.